R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -141,3 +141,26 @@ test_that("kmeans", {
   cluster <- summary.model$cluster
   expect_equal(sort(collect(distinct(select(cluster, "prediction")))$prediction), c(0, 1))
 })
+
+test_that("naiveBayes", {
+  training <- suppressWarnings(createDataFrame(sqlContext, iris))
```

Comment:
Then we can create a really small dataset with 3 categories and some count data. We can also verify against e1071::naiveBayes output.